rosidl_generator_py/cmake/rosidl_generator_py_generate_interfaces.cmake

```diff
@@ -273,6 +273,8 @@ foreach(_typesupport_impl ${_typesupport_impls})
   ament_target_dependencies(${_target_name}
     "rosidl_generator_c"
     "rosidl_generator_py"
+  )
+  target_link_libraries(${_target_name}
```

Comment:
I'm not 100% sure this needs to go through `target_link_libraries()` since it works currently, and there's an `add_dependencies()` call above.

Reply:
I removed it since it wasn't needed before.